types/src/Experiment/enums.ts

```diff
@@ -4,6 +4,10 @@ export enum CONSISTENCY_RULE {
   GROUP = 'group',
 }
 
+export enum DESIGN_TYPE {
```

Comment:
This is just placeholder, I want an enum for this value but I don't know if we'll need anything for this value that coordinates with backend, can be change later.